deepchem/dock/docking.py

```diff
@@ -1,119 +1,96 @@
-    call((
-        "wget -nv -c http://deepchem.io.s3-website-us-west-1.amazonaws.com/trained_models/random_full_RF.tar.gz"
-    ).split())
-    call(("tar -zxvf random_full_RF.tar.gz").split())
-    call(("mv random_full_RF %s" % (self.base_dir)).split())
-    self.model_dir = os.path.join(self.base_dir, "random_full_RF")
-
-    # Fit model on dataset
-    model = SklearnModel(model_dir=self.model_dir)
-    model.reload()
-
-    self.pose_scorer = GridPoseScorer(model, feat="grid")
-    self.pose_generator = VinaPoseGenerator(
-        exhaustiveness=exhaustiveness, detect_pockets=detect_pockets)
-
-  def dock(self,
-           protein_file,
-           ligand_file,
-           centroid=None,
-           box_dims=None,
-           dry_run=False):
-    """Docks using Vina and RF."""
-    protein_docked, ligand_docked = self.pose_generator.generate_poses(
-        protein_file, ligand_file, centroid, box_dims, dry_run)
-    if not dry_run:
-      score = self.pose_scorer.score(protein_docked, ligand_docked)
-    else:
-      score = np.zeros((1,))
-    return (score, (protein_docked, ligand_docked))
-
-
-'''
-class VinaGridDNNDocker(object):
-  """Vina pose-generation, DNN-models on grid-featurization of complexes."""
-
-  def __init__(self, exhaustiveness=10, detect_pockets=False):
-    """Builds model."""
-    self.base_dir = tempfile.mkdtemp()
-    logger.info("About to download trained model.")
-    call((
-        "wget -nv -c http://deepchem.io.s3-website-us-west-1.amazonaws.com/trained_models/random_full_DNN.tar.gz"
-    ).split())
-    call(("tar -zxvf random_full_DNN.tar.gz").split())
-    call(("mv random_full_DNN %s" % (self.base_dir)).split())
-    self.model_dir = os.path.join(self.base_dir, "random_full_DNN")
-
-    # Fit model on dataset
-    pdbbind_tasks = ["-logKd/Ki"]
-    n_features = 2052
-    model = MultitaskRegressor(
-        len(pdbbind_tasks),
-        n_features,
-        dropouts=[.25],
-        learning_rate=0.0003,
-        weight_init_stddevs=[.1],
-        batch_size=64,
-        model_dir=self.model_dir)
-    model.reload()
-
-    self.pose_scorer = GridPoseScorer(model, feat="grid")
-    self.pose_generator = VinaPoseGenerator(
-        exhaustiveness=exhaustiveness, detect_pockets=detect_pockets)
+    self.pose_generator = pose_generator
+    self.featurizer = featurizer
+    self.scoring_model = scoring_model
 
   def dock(self,
-           protein_file,
-           ligand_file,
+           molecular_complex,
            centroid=None,
            box_dims=None,
-           dry_run=False):
-    """Docks using Vina and DNNs."""
-    protein_docked, ligand_docked = self.pose_generator.generate_poses(
-        protein_file, ligand_file, centroid, box_dims, dry_run)
-    if not dry_run:
-      score = self.pose_scorer.score(protein_docked, ligand_docked)
+           exhaustiveness=10,
+           num_modes=9,
+           num_pockets=None,
+           out_dir=None):
+    """Docks using Vina and RF.
```

Comment:
This docstring has a bunch of references to Vina and RF that need to be changed.

Reply:
Good catch. Fixed